client/packages/common/src/types/schema.ts

```diff
@@ -4598,7 +4598,6 @@ export type UnitVariantNode = {
   __typename: 'UnitVariantNode';
   itemId: Scalars['String']['output'];
   mostUsedVariantId: Scalars['String']['output'];
-  unitName?: Maybe<Scalars['String']['output']>;
```

Comment:
unitName was removed from API (since api wont' provide unitNames for all items, and thus unitName will be used from other api calls, like stock.item.unitName)